pkg/kclient/kclient_test.go

```diff
@@ -0,0 +1,119 @@
+package kclient
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ktesting "k8s.io/client-go/testing"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func TestGetPVCsFromSelector(t *testing.T) {
```

Comment:
Can these be moved under the volumes_test.go?

Reply:
Updated